src/scripts/iri_sources.py

```diff
@@ -0,0 +1,58 @@
+"""Get a list of version IRIs for sources"""
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Dict, List, Union
+
+from oaklib import get_adapter
+from oaklib.implementations import SqlImplementation
+
+HERE = Path(os.path.abspath(os.path.dirname(__file__)))
+SRC_DIR = HERE.parent
+PROJECT_ROOT = SRC_DIR.parent
+ONTO_DIR = PROJECT_ROOT / 'src' / 'ontology'
+
+
+
+def iri_docs(rel_paths: List[str], outpath_txt: Union[Path, str], outpath_md: Union[Path, str]):
+    """Get a list of version IRIs for sources"""
+    version_curies: List[str] = []
```

Comment:
version_curies sounds odd

Reply:
Not sure what else to name it. `owl_version_curies`?

Reply:
i would leave this as `versions`

Reply:
Mm, OK. The reason I didn't do that is to disambiguate. Later on, I have `version_iris`. If I change them both to `versions`. But that is also a cleaner naming.